Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1505,25 +1505,59 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
 
 /// Performs matrix multiplication with another tensor and produces the result.
 @inlinable
-@differentiable(vjp: _vjpMatmul(_:_:) where Scalar: TensorFlowFloatingPoint)
+@differentiable(vjp: _vjpMatmul(_:_:transposeA:transposeB:) where Scalar: TensorFlowFloatingPoint)
 public func matmul<Scalar: Numeric>(
```

Comment:
How about calling it `matmul(_:transposed:_:transposed)` where the two `transposed`s have a default value?

Reply:
Can you call this like `matmul(x, y)` or do the arguments need to be ordered?

Reply:
Yep

Reply:
That's neat. I just pushed this change.